src/components/TableData/Row.styles.js

```diff
@@ -0,0 +1 @@
+export const labelStyle = { color: "rgba(0, 0, 0, 0.5)", fontSize: "14px" }
```

Comment:
Here is a missing new line too. Also I think that the file should be called TableRowDelivery.styles.js as there is no Row component and should be placed under the components folder?

```suggestion
export const labelStyle = { color: "rgba(0, 0, 0, 0.5)", fontSize: "14px" }

```

Reply:
Another suggestion is to place this line of code inside the component that is using it, given that it would be too much to create a file for a single line of code

Reply:
good good.